protocol/messenger_communities.go

```diff
@@ -5021,7 +5014,6 @@ func (m *Messenger) rekeyAllCommunities(logger *zap.Logger) {
 		//  c.RekeyedAt().Add(rki).Before(time.Now())
 		//  keyTimestampID + rki < time.Now()
 		//  Just using the vars that will be used later
```

Comment:
With this PR the rekey loop has been enabled with a default interval of 5 minutes. Without this segment of code, the community would rekey every 5 minutes, which I believe is excessively frequent. The intention was to have a rekeying process every hour (refer to line 4996), but to initiate a check on what to rekey every 5 minutes (see line 4966).

Reply:
We could keep the loop disabled and address it as followup to not block this PR anymore.

Reply:
oh I see, I will add a flag to disable and have it disabled for now, yes, good catch